extensions/ql-vscode/src/remote-queries/view/remoteQueries.css

```diff
@@ -1,5 +1,93 @@
-.vscode-codeql__result-table {
-  display: table;
-  border-collapse: collapse;
-  width: 100%;
+body {
+  font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Helvetica, Arial,
+    sans-serif, Apple Color Emoji, Segoe UI Emoji;
+}
+
+p {
+  margin: 0.5em 0 0.5em 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+.download-link {
```

Comment:
I think it's best to keep using the prefixes. 

The css in the original view uses [BEM style css](http://getbem.com/introduction/) in order to avoid naming conflicts. I'm not a fan of this convention anymore since there are now some better solutions, but they require using preprocessors in different ways.

This would be a nice clean-up task, to convert to sass or something similar. But until then, I think we should stick with the current approach.

Reply:
Thank you both! I've made the changes to stick to the current convention (I think!) but I'm a bit unclear why the prefixes are needed (do all styles go into a global space, not encapsulated within the React component?).

Agree it'd be cool to start using something like LESS (which seems to be configured already, looking at webpack.config.ts) when we get the chance.

Reply:
I'm pretty sure (but haven't checked recently that all css is in the global namespace. Currently, each webview only has a single css file so it's not really a big deal in practice.